core/src/test/java/org/jobrunr/server/carbonaware/CarbonAwareJobManagerTest.java

```diff
@@ -0,0 +1,247 @@
+package org.jobrunr.server.carbonaware;
+
+import org.jobrunr.jobs.Job;
+import org.jobrunr.jobs.states.CarbonAwareAwaitingState;
+import org.jobrunr.scheduling.carbonaware.CarbonAwarePeriod;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import static java.time.Instant.now;
+import static java.time.temporal.ChronoUnit.HOURS;
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.within;
+import static org.jobrunr.JobRunrAssertions.assertThat;
+import static org.jobrunr.jobs.JobTestBuilder.aJob;
+import static org.jobrunr.jobs.JobTestBuilder.anEnqueuedJob;
+import static org.jobrunr.jobs.states.StateName.AWAITING;
+import static org.jobrunr.jobs.states.StateName.SCHEDULED;
+import static org.jobrunr.server.carbonaware.CarbonApiMockResponses.BELGIUM_2024_07_11;
+import static org.jobrunr.server.carbonaware.CarbonApiMockResponses.BELGIUM_PARTIAL_2024_07_11_FULL_2024_07_12;
+import static org.jobrunr.server.carbonaware.CarbonApiMockResponses.BELGIUM_PARTIAL_2024_07_12;
+import static org.jobrunr.server.carbonaware.CarbonApiMockResponses.GERMANY_2024_07_11;
+import static org.jobrunr.server.carbonaware.CarbonApiMockResponses.UNKNOWN_AREA;
+import static org.mockito.DatetimeMocker.mockZonedDateTime;
+import static org.mockito.InstantMocker.mockTime;
+import static org.mockito.Mockito.clearInvocations;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class CarbonAwareJobManagerTest extends AbstractCarbonAwareWiremockTest {
+    @Test
+    void testGetTimezoneReturnsSystemZoneIdByDefault() {
+        assertThat(getCarbonAwareJobManager("DE").getTimeZone()).isEqualTo(ZoneId.systemDefault());
+    }
```

Comment:
This test will fail for users outside of CEST.

Reply:
Since no data is retrieved, the zonedId returned will be the system default. I changed the `areaCode` to `UNKNOWN` so it's less confusing.